wordpress-mobile/AztecEditor-iOS
# Release/1.12.0

Creating 1.12.0 Aztec-iOS release

### This release includes

* [Update demo for dark mode #1223](https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1223)
* [CircleCI: Bump Orbs to use any 1.0.x version #1227](https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1227)
* [Recalculate typing attributes on alignment update #1228](https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1228)
* [Make pre and quote background colors optional #1229](https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1229)

### To test

Build and run wordpress-mobile/WordPress-iOS#**TODO**
Check that the improvements referenced behave properly.
